index.html

```diff
@@ -1,216 +1,236 @@
       </div>
-    </section>
-    <section class="about-section">
-      <div class="container">
-        <h1>About<br />Myself</h1>
-        <p>
-          I’m a software developer! I can help you build a product , feature or
-          website Look through some of my work and experience! If you like what
-          you see and have a project you need coded, don’t hestiate to contact
-          me.
+      <p>
+        A daily selection of privately personalized reads; no accounts or
+        sign-ups required.
+      </p>
+      <ul class="programing-lamguages">
+        <li>html</li>
+        <li>css</li>
+        <li>javaScript</li>
+      </ul>
+      <button type="submit">See Project</button>
+    </div>
+  </section>
+  <section class="about-section">
+    <div class="container">
+      <h1>About<br />Myself</h1>
+      <p>
+        I’m a software developer! I can help you build a product , feature or
+        website Look through some of my work and experience! If you like what
+        you see and have a project you need coded, don’t hestiate to contact
+        me.
+      </p>
+    </div>
+    <div class="contact">
+      <a href="">
+        <p>LET'S CONNECT</p>
+      </a>
+      <ul>
+        <li>
+          <a href="https://twitter.com/RebbouhElseddik"><img src="./img/twitter.svg" alt="" /></a>
+        </li>
+        <li>
+          <a href=""><img src="./img/Vector(1).svg" alt="" /></a>
+        </li>
+        <li>
+          <a href=""><img src="./img/Vector(2).svg" alt="" /></a>
+        </li>
+        <li>
+          <a href="https://github.com/MohammedElseddik">
+            <img src="./img/github.svg" alt="" /></a>
+        </li>
+        <li>
+          <a href=""><img src="./img/Vector(4).svg" alt="" /></a>
+        </li>
+      </ul>
+      <button type="submit">Get my resume</button>
+    </div>
+
+    <div class="skill-list">
+      <ul class="categories-list">
+        <li>
+          <div>
+            <h2>langguages</h2>
+            <img class="arrow1" src="./img/arrow.svg" alt="" />
+          </div>
+          <ul class="langguages">
+            <li>
+              <img src="./img/javascript-icon.svg" alt="" />
+              <p>JavaScript</p>
+            </li>
+            <li>
+              <img src="./img/Html-icon.svg" alt="" />
+              <p>HTML</p>
+            </li>
+            <li>
+              <img src="./img/css-icon.svg" alt="" />
+              <p>CSS</p>
+            </li>
+          </ul>
+        </li>
+        <li>
+          <div>
+            <h2>Frameworks</h2>
+            <img class="arrow2" src="./img/arrow2.svg" alt="" />
+          </div>
+          <ul class="frameworks"></ul>
+        </li>
+        <li>
+          <div>
+            <h2>Skills</h2>
+            <img class="arrow2" src="./img/arrow2.svg" alt="" />
+          </div>
+          <ul class="skills"></ul>
+        </li>
+      </ul>
+    </div>
+  </section>
+
+  <section class="contact-form-section">
```

Comment:
- [ ] Instead of `<section>` element here, Please use `<footer>` tag. It will make sure your code is semantic and follows HTML best practices.